featuretools/tests/computational_backend/test_feature_set_calculator.py

```diff
@@ -69,8 +69,8 @@ def test_make_dfeat(es):
     assert (v == 33)
 
 
-def test_make_agg_feat_of_identity_column(es):
-    agg_feat = ft.Feature(ft.Feature(es, 'log', 'value'), parent_dataframe_name='sessions', primitive=Sum)
+def test_make_agg_feat_of_identity_variable(es):
+    agg_feat = ft.Feature(ft.Feature(es['log'].ww['value']), parent_dataframe_name='sessions', primitive=Sum)
```

Comment:
I see a lot of calls like this, but I'm not sure we need the inner `ft.Feature()` call now? We needed them before this update, but I'm thinking we don't now? This could become:

```
agg_feat = ft.Feature(es['log'].ww['value'], parent_dataframe_name='sessions', primitive=Sum)
```

If these nested `ft.Feature` calls aren't needed any more, I think we should remove them before merging this.